### DISCUSSION TITLE
Duvida sobre Class Library

### DISCUSSION THREAD

**Discussion by RonaldoBueno16** [balta-io/forum#323] [Tira Dúvidas] [answered]
É possível utilizar o tipo de projeto Class Library e em vez de compilar para .dll, compilar para .so para utilizar em Linux?

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @RonaldoBueno16, como vai?

Até onde sei, não, o resultado de uma class Library é sempre uma DLL.
Você pode ter projetos do tipo Console e gerar um pacote deles no final, mas o Class Library, somente DLL.

Mas vale a pena uma pesquisa mais funda, se achar algo, posta aqui que dou minha opinião.

Abraços.
